lib/content_item_loader.rb

```diff
@@ -43,7 +43,26 @@ def load_from_sources(base_path)
   end
 
   def use_graphql?
-    request&.params&.[]("graphql") == "true"
+    return false unless request
+
+    if request.params["graphql"] == "true"
+      return true
+    elsif request.params["graphql"] == "false"
+      return false
+    end
+
+    if request.headers
```

Comment:
There isn't a test for if request.headers is nil (we also appear to have lost coverage of line 36 above, testing if there's a graphql response, but it's not in the allowed schemas?

Reply:
I've added a test for headers being `nil` and also ensured we're testing on the B variant for schemas that aren't allowed.

Reply:
Hmm - this line is still showing as only being covered for the case when request.headers is not nil. (I can see that line 36 is uncovered in the main branch though, so I'll take care of that, but you might have to do a small rebase)

Reply:
Looks like I never pushed the branch after adding the test for the header being nil 🤦‍♂️.

Reply:
Thanks!